feature_reviser/transformer/string_transformer.py

```diff
@@ -279,3 +279,89 @@ def __phone_to_float(
                 return float(re.sub(r"(?<!^)[^0-9]", "", error_value))
             except:  # pylint: disable=W0702
                 return float(error_value)
+
+
+class StringTruncatorTransformer(BaseTransformer):
+    """
+    Truncates each entry in a string column up to a specified number of characters.
+
+    Args:
+        feature (str): The feature which should be transformed.
+        n_chars (int): The number of characters to truncate (default = 1).
+    """
+
+    def __init__(
+        self,
+        feature: str,
+        n_chars: int = 1,
+    ) -> None:
+        self.feature = feature
+        self.n_chars = n_chars
+
+    def transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Truncates a string feature up to a specified number of characters.
+
+        Args:
+            X (pandas.DataFrame): DataFrame to transform.
+
+        Returns:
+            pandas.DataFrame: Original dataframe with truncated strings in the feature.
+        """
+
+        X = check_X(X)
+
+        X[self.feature] = [x[: self.n_chars] for x in X[self.feature]]
+
+        return X
+
+
+class StringSlicerTransformer(BaseTransformer):
+    """
+    Slices all entries of specified string features using the `slice()` function.
+
+    Note: The arguments for the `slice()` function are passed as a tuple. This shares the python quirk of writing a tuple with a single argument with the trailing comma.
+
+    Example:
+        >>> from feature_reviser import StringSlicerTransformer
+        >>> import pandas as pd
+        >>> X = pd.DataFrame({"foo": ["abc", "def", "ghi"], "bar": ["jkl", "mno", "pqr"]})
+        >>> transformer = StringSlicerTransformer([("foo", (0, 3, 2)), ("bar", (2,))])
+        >>> transformer.fit_transform(X).values
+        array([['ac', 'jk'],
+               ['df', 'mn'],
+               ['gi', 'pq']], dtype=object)
+
+    Args:
+        slice_args (List[Tuple[str, Tuple[int, int, int]]]): The arguments to the `slice` function, for each feature.
+    """
+
+    def __init__(
+        self,
+        slice_args: List[
```

Comment:
I propose renaming this to `features` since in most other functions is has the same name. What do you think?

Reply:
Resolved in the commit https://github.com/chrislemke/feature-reviser/pull/7/commits/bfc1a3bc7552f012f905928d4cd1da045996b82a